.github/dependabot.yml

```diff
@@ -17,10 +17,3 @@ updates:
     commit-message:
       prefix: chore
       include: scope
-    ignore:
```

Comment:
I'm not sure if we can remove these but I want to see how dependabot behaves after this PR gets merged.